### PR TITLE
stub api login

### PR DESCRIPTION
**Pull request by springerkc**

### CODE REVIEW

File: apps/operator/src/lib/auth.ts [hunk 1]
```diff
@@ -25,18 +26,33 @@ export const config = {
...
+        if (fData.ok) {
+          const data = await fData.json()
+
+          /**
+           * Should we be calling the /v1/authenticate endpoint after
+           * successful login to get an access token so its a passes
+           * though to our auth callbacks below?
+           *
+           * This call will return a 200 { message: success } if
+           * login is successful, no auth token
```
Comment (golanglemonade) [inline apps/operator/src/lib/auth.ts:48]: The login endpoint returns the access and refresh tokens in the cookies in the response, which should be able to be used in subsequent requests

File: apps/operator/src/app/(auth)/signup/page.tsx [hunk 2]
```diff
@@ -22,15 +27,21 @@ const Landing: React.FC = () => {
...
+              <TextInput name="last_name" placeholder="Baggins" />
+            </div>
+            <TextInput name="username" placeholder="email@domain.net" />
             <TextInput name="password" placeholder="password" type="password" />
             <TextInput
-              name="confirmPassword"
+              name="confirmedPassword"
               placeholder="confirm password"
               type="password"
             />
```
Comment (golanglemonade) [inline apps/operator/src/app/(auth)/signup/page.tsx:47]: The Button text (Iine 49)  says `Sign In` but should say `Register` or `Sign Up`

File: apps/operator/src/lib/auth.ts (hunk 1 above)
Comment (springerkc) [reply apps/operator/src/lib/auth.ts:48]: Yea good call I saw that, just figured if you said some day it wasn't we should just build the thing to get it in the callbacks via a `/verify` or `/authenticate` call. happy to just write it around the token being here for now though!

File: packages/dally/src/lib/user.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  }
+}
+
+export const useRegister = (
+  payload: {
+    first_name: string
+    last_name: string
+    email: string
+    password: string
+    confirmedPassword: string
```
Comment (golanglemonade) [inline packages/dally/src/lib/user.ts:42]: What is confirming the passwords match? I don't see this used anywhere?

Comment (springerkc) [reply packages/dally/src/lib/user.ts:42]: I didnt write it yet, but its such common practice I figured on before submit we would just validate they're they same real quick. I can add it real quick if you want.

Comment (golanglemonade) [reply packages/dally/src/lib/user.ts:42]: nah, that can be a follow-up, was just checking I wasn't missing it

**Review by golanglemonade — APPROVED** [review]

**Review by golanglemonade — APPROVED** [review]

**Pull request merged**